### PR TITLE
feat: NPG-1368 drep incentives tests

### PR DESCRIPTION
**Pull request by nicopado**

### CODE REVIEW

**Review by stevenj — APPROVED** [review]
Looks OK to me.

**Comment by minikin** [comment]
@nicopado Can we fix conflicts/ci and merge this PR, please?

File: src/vit-testing/integration-tests/src/e2e/rewards/drep_rewards.rs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+#[test] // no direct voter, 4 dreps with equal amount of voting power
+pub fn drep_rewards_happy_path() {
+    let rep_voting_group: VotingGroup = "rep".to_string();
+
+    let testing_directory = TempDir::new().unwrap().into_persistent();
+
+    let drep1_wallet = CardanoWallet::new(1000);
+    let drep2_wallet = CardanoWallet::new(1000);
+    let drep3_wallet = CardanoWallet::new(1000);
+    let drep4_wallet = CardanoWallet::new(1000);
```
Comment (kukkok3) [inline src/vit-testing/integration-tests/src/e2e/rewards/drep_rewards.rs:42]: magic numbers:)

Comment (nicopado) [reply src/vit-testing/integration-tests/src/e2e/rewards/drep_rewards.rs]: solved! :D

**Review by kukkok3 — APPROVED** [review]

File: src/vit-testing/integration-tests/src/e2e/rewards/drep_rewards.rs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        } else {
+            snapshot_doctored_copy.push(voter_info);
+        }
+    }
+
+    let vote_timing = VoteBlockchainTime {
+        vote_start: 0,
+        tally_start: 1,
+        tally_end: 2,
+        slots_per_epoch: 30,
```
[the remark this answers is not in the document]
Comment (nicopado) [reply src/vit-testing/integration-tests/src/e2e/rewards/drep_rewards.rs]: Thanks! Changed as requested (this is not run in CI currently however, still it's better to correct it now for the future)

**Pull request merged**